core/bones/json.py

```diff
@@ -53,7 +55,9 @@ def unserialize(self, skel: 'viur.core.skeleton.SkeletonInstance', name: str) ->
 
         return False
 
-    def singleValueFromClient(self, value: Union[str, list, dict], *args, **kwargs):
+    def singleValueFromClient(self, value: Union[str, list, dict], skel: 'SkeletonInstance',
+                              bone_name: str, client_data: dict
+                              ) -> tuple[Any, list[ReadFromClientError] | None]:
```

Comment:
Is it necessary to provide full type annotations in sub-classes as well?
I would like to keep maintenance overhead low, if possible.